# SONAR_DISTANCE message data

Header header

float32 sonar_front
float32 sonar_behind
float32 sonar_left
float32 sonar_right
float32 sonar_up
float32 sonar_down
float32 sonar_cam

